psst-gui/src/ui/credits.rs

```diff
@@ -0,0 +1,218 @@
+use std::sync::Arc;
+
+use crate::{
+    cmd,
+    data::{AppState, ArtistLink, Nav},
+    ui::theme,
+    ui::utils,
+};
+use druid::{
+    widget::{Controller, CrossAxisAlignment, Flex, Label, List, Maybe, Painter, Scroll},
+    Cursor, Data, Env, Event, EventCtx, Lens, RenderContext, Target, UpdateCtx, Widget, WidgetExt,
+};
+use serde::Deserialize;
+
+#[derive(Debug, Clone, Data, Lens, Deserialize)]
+pub struct TrackCredits {
+    #[serde(rename = "trackUri")]
```

Comment:
Instead using this we should use: #[serde(rename_all = "camelCase")]

Reply:
Thanks!